src/Conda.jl

```diff
@@ -103,9 +103,55 @@ function _set_conda_env(cmd, env::Environment=ROOTENV)
     env_var["PYTHONIOENCODING"]="UTF-8"
     env_var["CONDARC"] = conda_rc(env)
     env_var["CONDA_PREFIX"] = prefix(env)
+    _set_path(env_var, env)
     setenv(cmd, env_var)
 end
 
+const path_sep = Compat.Sys.iswindows() ? ';' : ':'
+
+function _set_path(env_var, env::Environment)
+    if Compat.Sys.iswindows()
+        # Environment variables are case-insensitive.  `Base.EnvDict`
+        # treats case sensitivity but `env_var` is an arbitrary
+        # dicttionary.  So, we need to treat case-insensitive lookup
+        # ourselves:
+        all_keys = collect(keys(env_var))
+        ikey = findfirst(x -> uppercase(x) == "PATH", all_keys)
+        has_key = ikey !== nothing
+        path_key = all_keys[ikey]
+    else
+        path_key = "PATH"
+        has_key = haskey(env_var, path_key)
+    end
+
+    if has_key
+        env_var[path_key] = extra_path(env) * path_sep * env_var[path_key]
+    else
+        env_var[path_key] = extra_path(env)
+    end
+end
+
+"""
+    extra_path(env::Environment)
+
+Extra `ENV["PATH"]` required for conda environment `env`.
+"""
+function extra_path(env::Environment)
```

Comment:
Somebody has to write a code to pass this to SetDllDirectory or AddDllDirectory in `PyCall.__init__`: https://github.com/JuliaPy/Conda.jl/issues/125#issuecomment-423847333